octoprint_TimeToFilament/__init__.py

```diff
@@ -19,6 +21,8 @@ class TimeToFilamentPlugin(octoprint.plugin.SettingsPlugin,
                            octoprint.plugin.StartupPlugin,
                            octoprint.plugin.BlueprintPlugin):
 
+  variables = dict()
```

Comment:
This should be initialized in `__init__` and be called `self.variables`, otherwise, all instances of this class will share the same data member.  Can you please do this?

Reply:
ah, this is a leftover from a version before :-(